src/app/clusters/ota-provider/ota-provider.cpp

```diff
@@ -226,6 +226,13 @@ namespace app {
 namespace Clusters {
 namespace OTAProvider {
 
+void ResetDelegate()
```

Comment:
If we need to zero out the array, we should just do that in the cluster's init method, instead of forcing all consumers to call this method (which would be broken if there were ever > 1 OTA endpoint).

Reply:
agree, updated, thanks